norfair/tracker.py

```diff
@@ -10,12 +10,12 @@
 class Tracker:
     def __init__(
         self,
-        distance_function,
-        distance_threshold,
-        hit_inertia_min=10,
-        hit_inertia_max=25,
-        detection_threshold=0,
-        point_transience=4,
+        distance_function: Callable,
+        distance_threshold: Union[int, float],
```

Comment:
The `Union` is not necessary, `float` includes `int`s as described [here](https://twitter.com/llanga/status/1306391094847057920). Thanks @erovira for the tip!